### ISSUE TITLE
OSX 10.11 Beta (15A244d) 6 - unable to load kext

### ISSUE THREAD

**Issue by S8ER01Z** [tekezo/Karabiner#450]
I am getting a message at boot about the kext not loading then this in console...
"8/4/15 8:44:47.236 AM Karabiner[534]: [INFO] IOService is not found.
 8/4/15 8:44:48.621 AM Karabiner[534]: [INFO] BRIDGE_USERCLIENT_SYNCHRONIZED_COMMUNICATION connection is null"

I loaded the kext... without any issues...  
sudo /Applications/Karabiner.app/Contents/Library/bin/kextload load <- Worked

Thoughts?  If you need anything to sort this out please let me know.

**Comment by fizz** [comment]
:+1:  manually loading the kext made it work for me.

**Comment by caidaoli** [comment]
:+1: manually loading the kext made it work for me.

**Comment by zchee** [comment]
FYI,
```
kextload /Applications/Karabiner.app/Contents/Library/Karabiner.signed.kext
```

**Comment by neilio** [comment]
Thanks for the manual fix. Same issue here, also worked around by manually loading the kext.

**Comment by christopherdwhite** [comment]
Thanks guys! It's amazing how much you release you use something when it's gone. Amazing utility!

**Comment by tekezo** [comment]
I've fixed this issue at v10.6.92. (The latest beta version)

How to upgrade to the latest beta version:
https://pqrs.org/osx/karabiner/document.html#check-for-updates
